app/src/main/java/com/duckduckgo/app/browser/logindetection/FireproofDialogsEventHandler.kt

```diff
@@ -39,19 +38,21 @@ interface FireproofDialogsEventHandler {
     suspend fun onDisableLoginDetectionDialogShown()
     suspend fun onUserConfirmedDisableLoginDetectionDialog()
     suspend fun onUserDismissedDisableLoginDetectionDialog()
+    suspend fun onUserEnabledAutomaticFireproofLoginDialog(domain: String)
+    suspend fun onUserFireproofSiteAutomaticFireproofLoginDialog(domain: String)
```

Comment:
Likewise with this one, maybe: `onUserRequestedAskEverytime()` or something like that?